float32 h_angle # 0.0 <= h_angle <= 360.0
float32 v_angle # 0.0 <= v_angle <= 360.0
string emotion
